### PR TITLE
feat: learner credit mgmt offer metadata display

### PR DESCRIPTION
**Pull request by adamstankiewicz**
# Screenshots

image

image

image

# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/data/services/EnterpriseDataApiService.js [hunk 1]
```diff
@@ -27,6 +30,38 @@ class EnterpriseDataApiService {
...
 
+  static fetchEnterpriseOfferUtilization(enterpriseUUID, offerId, options = {}) {
+    const queryParams = new URLSearchParams({
+      ...snakeCaseObject(options),
+    });
+    // eslint-disable-next-line no-unused-vars
+    const url = `${EnterpriseDataApiService.enterpriseBaseUrl}${enterpriseUUID}/offers/${offerId}/?${queryParams.toString()}`;
+
+    // mocked api response
+    return new Promise((resolve) => {
```
Comment (adamstankiewicz) [inline src/data/services/EnterpriseDataApiService.js:41]: temporarily mocked out API for aggregate offer data on utilization

File: src/components/learner-credit-management/LearnerCreditAllocationTable.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          Cell: ({ row }) => moment(row.values.enrollmentDate).format('MMMM DD, YYYY'),
+          disableFilters: true,
+        },
+      ]}
+      initialTableOptions={{
+        getRowId: row => row.enterpriseEnrollmentId.toString(),
+      }}
+      initialState={{
+        pageSize: PAGE_SIZE,
+        pageIndex: DEFAULT_PAGE - 1, // `DataTable` uses zero-indexed array
```
Comment (long74100) [inline src/components/learner-credit-management/LearnerCreditAllocationTable.jsx:63]: should `DEFAULT_PAGE` be set to 0 and the comment moved up there instead?

File: src/components/learner-credit-management/LearnerCreditManagement.jsx [hunk 3]
```diff
@@ -1,15 +1,88 @@
...
-      <title>Learner Credit Management</title>
-    </Helmet>
-    <Hero title="Learner Credit Management" />
-  </>
-);
+const LearnerCreditManagement = ({ enterpriseUUID }) => {
+  const { offers } = useContext(EnterpriseSubsidiesContext);
 
-export default LearnerCreditManagement;
+  const enterpriseOffer = useMemo(() => [...offers].shift(), [offers]);
```
Comment (long74100) [inline src/components/learner-credit-management/LearnerCreditManagement.jsx:22]: nit: can we use just user offers[0] here?

**Review by long74100 — APPROVED** [review]

File: src/components/learner-credit-management/TableTextFilter.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import PropTypes from 'prop-types';
+import { Form, Icon } from '@edx/paragon';
+import { Search } from '@edx/paragon/icons';
+
+const TableTextFilter = ({
+  column: {
+    filterValue, setFilter, Header,
+  },
+}) => {
+  const inputText = `Search by ${Header.toLowerCase()}`;
```
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply src/components/learner-credit-management/TableTextFilter.jsx:11]: It sure does ;) And in the default `TextFilter` that `@edx/paragon` ships with!

**Pull request merged**